base-java/Dockerfile

```diff
@@ -1,9 +1,8 @@
 ARG JITSI_REPO=jitsi
 FROM ${JITSI_REPO}/base
 
-RUN	\
-	mkdir -p /usr/share/man/man1 && \
-	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -y openjdk-8-jre-headless && \
-	apt-cleanup
-
+RUN mkdir -p /usr/share/man/man1 && \
+    apt-dpkg-wrap apt-get update && \
+    apt-dpkg-wrap apt-get install -y openjdk-8-jre-headless && \
+    apt-cleanup && \
+    rm -rf /tmp/pkg /var/cache/apt
```

Comment:
Nothing creates /tmp/pkg here, no need to remove it. Also, how about deleting /var/cache/apt in apt-cleanup?

Reply:
The actual useless path there is `/tmp/pkg`. Removing `/var/cache/apt` is just a more radical version of `apt-get clean` (too much of one?) and is already used in combination with it on the master branch:
https://github.com/jitsi/docker-jitsi-meet/blob/b039b29b7aeeee28f9f7395ec87e8aa69105ce9f/web/Dockerfile#L17
I was rather conservative in the approach I took, but you're right: anything related to APT cleanup shall be moved to the cleaning script.